src/TableTransforms.jl

```diff
@@ -57,6 +57,7 @@ export
   Replace,
   Coalesce,
   Coerce,
+  dropExtrema,
```

Comment:
Notice that we have a naming convention that needs to be followed. The transforms always start with a capital letter.

Reply:
I will update it sir , thanks for pointing out